core/OrderAnalyser.py

```diff
@@ -40,11 +40,19 @@ def start(self):
 
         if not launched_synapses:
             Utils.print_info("No synapse match the captured order: %s" % self.order)
-        else:
-            for synapse in launched_synapses:
-                params = self._get_synapse_params(synapse, self.order)
-                for neuron in synapse.neurons:
-                    self._start_neuron(neuron, params)
+
+        if not launched_synapses and self.main_controller.settings.default_synapse is not None:
+            default_synapse = self._get_default_synapse(self.brain.synapses, self.main_controller.settings.default_synapse)
+
+            if default_synapse is not None:
```

Comment:
1st test is testing the option in settings (as it's optional), second one checks if the synapse defined in settings has been found in the list of all synapse